fusedrug/data/tokenizer/modulartokenizer/modular_tokenizer.py

```diff
@@ -1082,9 +1089,15 @@ def encode_list(
                     f"Encountered unknown tokens in input starting with {typed_input_list[0].input_string}"
                 )
             elif on_unknown == "warn":
-                warn(
-                    f"Encountered unknown tokens in input starting with {typed_input_list[0].input_string}"
-                )
+                if verbose <= 1:
+                    warning_message = "Encountered unknown tokens in input"
+                elif verbose == 2:
+                    warning_message = f"Encountered unknown tokens in input starting with {typed_input_list[0].input_string[:20]}"
```

Comment:
Printing the token[s] can be helpful.  

One option that may be useful is to print each unknown token about once.  We do want to know what the unknown tokens are, but we need to not flood the logs.

Reply:
Also I see the number of unknown tokens is known so maybe print that? X/Y unknown tokens

Reply:
> Also I see the number of unknown tokens is known so maybe print that? X/Y unknown tokens

Added this on verbose 2 and 3